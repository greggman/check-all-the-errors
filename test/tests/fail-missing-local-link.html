<a href="resources/no-exists.html"></a>