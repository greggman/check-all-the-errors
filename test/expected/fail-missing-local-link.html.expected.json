{
  "numErrors": 2,
  "pages": {
    "http://localhost:8090/fail-missing-local-link.html": {
      "status": 200,
      "errors": [
        {
          "type": "badlink",
          "link": "http://localhost:8090/resources/no-exists.html",
          "status": 404
        }
      ]
    },
    "http://localhost:8090/resources/no-exists.html": {
      "status": 404,
      "errors": [
        {
          "type": "msg",
          "location": {
            "url": "http://localhost:8090/resources/no-exists.html"
          },
          "text": "Failed to load resource: the server responded with a status of 404 (Not Found)"
        }
      ]
    }
  },
  "responses": [
    {
      "href": "http://localhost:8090/fail-missing-local-link.html",
      "status": 200
    },
    {
      "href": "http://localhost:8090/resources/no-exists.html",
      "status": 404
    }
  ]
}